benchmark/single-source/CSVParsing.swift

```diff
@@ -1,7 +1,25 @@
 import TestsUtils
-public let CSVParsing = BenchmarkInfo(
```

Comment:
Is there a particular reason to add 4 separate entries instead of switching to an array, like this?
````swift
public let CSVParsing = [
 BenchmarkInfo(name: "CSVParsing.Char",
    runFunction: run_CSVParsing,	    runFunction: run_CSVParsing_characters,
    tags: [.miniapplication, .api, .String],
    setUpFunction: { buildWorkload() },
    legacyFactor: 11),
BenchmarkInfo(
    name: "CSVParsing.Scalar",
    runFunction: run_CSVParsing_scalars,
    tags: [.miniapplication, .api, .String],
    setUpFunction: { buildWorkload() },
    legacyFactor: 11),
BenchmarkInfo(
    name: "CSVParsing.UTF16",
    runFunction: run_CSVParsing_utf16,
    tags: [.miniapplication, .api, .String],
    setUpFunction: { buildWorkload() },
    legacyFactor: 11),
 BenchmarkInfo(
    name: "CSVParsing.UTF8",
    runFunction: run_CSVParsing_utf8,
    tags: [.miniapplication, .api, .String],
    setUpFunction: { buildWorkload() },
    legacyFactor: 11)
]